presto-product-tests/README.md

```diff
@@ -221,6 +221,7 @@ groups.
 | HDFS impersonation    | ``hdfs_impersonation``    | ``singlenode-hdfs-impersonation``, ``singlenode-kerberos-hdfs-impersonation``    |
 | No HDFS impersonation | ``hdfs_no_impersonation`` | ``singlenode``, ``singlenode-kerberos-hdfs-no_impersonation``                    |
 | LDAP                  | ``ldap``                  | ``singlenode-ldap``                                                              |
+| SQL Server            | ``sqlserver``             | ``singlenode-sqlserver``
```

Comment:
This is missing the `|` at the end of the line (not sure it's required but the others have it)